CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+- CASM-4350: Resolve 1MB size limitation of the `cray-product-catalog` Kubernetes ConfigMap by splitting
+  its content, storing each product's `component_versions` data in its own ConfigMap.
```

Comment:
IMO, this belongs under a "Changed" heading and not under "Added". Furthermore, I would suggest specifically naming what changes here, which is solely the update logic in `catalog_update.py`. With its current wording, it looks like all the changes required create, read, update, and delete the new split ConfigMap data have been implemented here, but that is not the case. Here's my suggested wording:

```suggestion
### Changed
- CASM-4350: To address the 1MiB size limit of Kubernetes ConfigMaps, the 
  `cray-product-catalog` Kubernetes ConfigMap is split into multiple smaller
  ConfigMaps with each product's `component_versions` data in its own ConfigMap.
  Modify the `catalog_update.py` script to update data across these ConfigMaps.
```